manageiq-api.gemspec

```diff
@@ -22,6 +22,10 @@ Gem::Specification.new do |spec|
   # 2.9.0 regressed serialization
   # See: https://github.com/rails/jbuilder/issues/461
   spec.add_dependency "jbuilder", "~> 2.5", "!= 2.9.0"
+  spec.add_dependency "multi_json"
+  spec.add_dependency "rails"
+  spec.add_dependency "responders"
+  spec.add_dependency "zeitwerk"
```

Comment:
Also here - not loving all the extra requirements (I realize this is WIP, so not really complaining yet)

Reply:
yeah, some were missing dependencies that are only obvious when I tested zeitwerk by enabling eager loading for nearly everything that was fairly easy to autoload.

I can look at extracting the missing dependencies and get them merged on master first.